### DISCUSSION TITLE
Is Kobo libra 2 Colour and Clara BW Supported?

### DISCUSSION THREAD

**Discussion by SilhouetteCipher** [koreader/koreader#11958] [Q&A]
Been having issues with KOReader crashing on boot on the Libra 2 Colour. I know the documentation says all kobo touch onwards are supported but is that correct? I just know the colour and Clara BW are newer so wasnt sure if they are supported yet.

Thanks

**Comment by Frenzie** [comment]
They are, but for one or two more weeks only in the nightlies. 

http://build.koreader.rocks/download/nightly/

**Comment by SilhouetteCipher** [comment]
thanks so much. Annoyed at myself i didnt think to check that!

**Comment by williamtsoi1** [comment]
I also have a Kobo Libra 2 Colour. Just wondering when the next release will come out so we don't have to install the nightly?

**Reply by Frenzie** [reply]
Probably soon but the nightly is very similarity. It's mainly things like translations being more finished and some extra testing at this point.
